source/manual/deployment-dashboards.html.md

```diff
@@ -27,6 +27,10 @@ These show/hide vertical lines on the other panels that show when events happene
 
 ![Deployment lines](images/deployment_dashboards/deployment_lines.png)
 
+Implementation note:
+
+Deployment lines use `events` data as normal graphite data is not available in all environments.
```

Comment:
Might be worth clarifying this slightly in case people don't know that event data is also part of Graphite. (Well, I didn't. 🙃) Maybe something like this?

> Deployment lines use graphite `events` data as deployment data in graphite's normal time-series format is not available in all environments.